worlds/witness/test/test_ep_shuffle.py

```diff
@@ -0,0 +1,67 @@
+from ..test import WitnessTestBase
+
+
+class TestIndividualEPs(WitnessTestBase):
+    options = {
+        "shuffle_EPs": "individual",
+        "EP_difficulty": "normal",
+        "obelisk_keys": True,
+        "disable_non_randomized_puzzles": True,
+        "shuffle_postgame": False,
+        "victory_condition": "mountain_box_short",
+        "early_caves": "off",
+    }
+
+    def test_eps_exist(self) -> None:
+        """
+        Test that EP locations exist in shuffle_EPs by checking for the existence of Tutorial First Hallway EP
+        """
+        self.assert_location_exists("Tutorial First Hallway EP")
```

Comment:
each `def` will roll a game, so it would be better to put all asserts into a single def. If you want them to show up separately and not abort after the first one, you can use subtests.

Reply:
I'm going to double check that, but if it's true, I agree and will change it

Reply:
Alright, fixed